src/module.ts

```diff
@@ -16,6 +17,23 @@ export default defineNuxtModule<AlgoliaOptions>({
       throw new Error('Missing `applicationId`')
     }
 
+    if (options.crawler) {
```

Comment:
What is missing here are the docs explaining how to use the crawler. Could you please add a information in this page about the new options and the types similarly to how it was done before? https://github.com/nuxt-modules/algolia/blob/main/docs/pages/2.setup.md